src/main/kotlin/org/rucca/cheese/model/TaskParticipantRealNameInfoDTO.kt

```
@@ -0,0 +1,49 @@
+    @get:JsonProperty("applyReason", required = true)
+    val applyReason: kotlin.String,
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("personalAdvantage", required = true)
+    val personalAdvantage: kotlin.String,
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("remark", required = true)
+    val remark: kotlin.String,
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("className")
+    val className: kotlin.String? = null
+) {}
```

detekt

[warning] The class or object TaskParticipantRealNameInfoDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)